specification/eventgrid/data-plane/readme.md

```diff
@@ -38,6 +38,51 @@ A sample valid event definition is shown below:
 ```
 ~~~
 
+In addition to the event schema definition, you must provide a JSON example of a real event the service can trigger. This example must be in a "examples" folder close to your JSON, and called using snakeCase based on the event final name. The example should contain the envelope, but could be CloudEvent or EventGrid schema, whatever is easier. No PR will be accepted without the example.
```

Comment:
Should we ask that the sample event be triggered by their actual code that will be used to publish the events? If they don't have this code yet, then they shouldn't submit the event until they do.

Reply:
I think it's fair to say, event must have been received at least from canary. Is that acceptable?

Reply:
Yeah that makes sense.